game-core/src/main/java/games/strategy/engine/data/properties/CollectionProperty.java

```diff
@@ -62,7 +63,7 @@ public void focusGained(final FocusEvent e) {}
 
       @Override
       public void focusLost(final FocusEvent e) {
-        // TODO: change m_values
+        // TODO: change values
```

Comment:
This comment should probably get removed as it doesn't make clear what needs to be done.

Reply:
b4aff8e72  The entire listener was a no-op, so I just removed it.